From 87e94e783237291c105fa77495d7903f3f476485 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 25 Aug 2017 17:52:03 +0800
Subject: [PATCH 2457/5242] MLK-16274 ARM64: dts: correct the pad setting of
 USDHC

commit  bcd32305938e344101369406975e1e7881872209 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8QM/i.MX8QX USDHC pads are dual voltage pads, and the
defiintation of each bit are different. Only bit[0] define
the drive strength slection, bit[4:1] are reserved.
  0 means high drive strength
  1 means low drive strength

This patch correct these pad setting, setting the usdhc
100mhz/200mhz pin at high drive strength.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |   76 +++++++++----------
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |   78 ++++++++++----------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |   74 +++++++++----------
 3 files changed, 114 insertions(+), 114 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 2864fb5..58783ab 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -301,35 +301,35 @@
 
 		pinctrl_usdhc1_100mhz: usdhc1grp100mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000045
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000025
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000025
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000025
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000025
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000025
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000025
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000025
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000025
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000025
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000045
-				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000021
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000020
 			>;
 		};
 
 		pinctrl_usdhc1_200mhz: usdhc1grp200mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000047
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000027
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000027
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000027
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000027
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000027
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000027
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000027
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000027
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000027
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000047
-				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000021
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000020
 			>;
 		};
 
@@ -355,25 +355,25 @@
 
 		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000045
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000025
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000025
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000025
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000025
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000025
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000020
 			>;
 		};
 
 		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000047
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000027
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000027
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000027
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000027
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000027
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000020
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 2a59f22..161dec3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -290,41 +290,41 @@
 
 		pinctrl_usdhc1_100mhz: usdhc1grp100mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000045
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000025
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000025
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000025
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000025
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000025
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000025
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000025
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000025
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000025
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000045
-				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000021
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000020
 			>;
 		};
 
 		pinctrl_usdhc1_200mhz: usdhc1grp200mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000047
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000027
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000027
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000027
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000027
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000027
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000027
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000027
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000027
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000027
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000047
-				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000021
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000020
 			>;
 		};
 
 		pinctrl_usdhc2_gpio: usdhc2gpiogrp {
 			fsl,pins = <
-				SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x06000048
+				SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x06000021
 				SC_P_USDHC1_WP_LSIO_GPIO4_IO21		0x06000021
 				SC_P_USDHC1_CD_B_LSIO_GPIO4_IO22	0x06000021
 			>;
@@ -344,25 +344,25 @@
 
 		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000045
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000025
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000025
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000025
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000025
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000025
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000020
 			>;
 		};
 
 		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000047
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000027
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000027
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000027
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000027
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000027
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000020
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index ad18609..732f9e6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -207,39 +207,39 @@
 
 		pinctrl_usdhc1_100mhz: usdhc1grp100mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000045
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000025
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000025
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000025
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000025
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000025
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000025
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000025
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000025
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000025
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000045
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
 			>;
 		};
 
 		pinctrl_usdhc1_200mhz: usdhc1grp200mhz {
 			fsl,pins = <
-				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000047
-				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000027
-				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000027
-				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000027
-				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000027
-				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000027
-				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000027
-				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000027
-				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000027
-				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000027
-				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000047
+				SC_P_EMMC0_CLK_CONN_EMMC0_CLK		0x06000040
+				SC_P_EMMC0_CMD_CONN_EMMC0_CMD		0x00000020
+				SC_P_EMMC0_DATA0_CONN_EMMC0_DATA0	0x00000020
+				SC_P_EMMC0_DATA1_CONN_EMMC0_DATA1	0x00000020
+				SC_P_EMMC0_DATA2_CONN_EMMC0_DATA2	0x00000020
+				SC_P_EMMC0_DATA3_CONN_EMMC0_DATA3	0x00000020
+				SC_P_EMMC0_DATA4_CONN_EMMC0_DATA4	0x00000020
+				SC_P_EMMC0_DATA5_CONN_EMMC0_DATA5	0x00000020
+				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000020
+				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000020
+				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x06000040
 			>;
 		};
 
 		pinctrl_usdhc2_gpio: usdhc2gpiogrp {
 			fsl,pins = <
-				SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x06000048
+				SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x06000021
 				SC_P_USDHC1_WP_LSIO_GPIO4_IO21		0x06000021
 				SC_P_USDHC1_CD_B_LSIO_GPIO4_IO22	0x06000021
 			>;
@@ -259,25 +259,25 @@
 
 		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000045
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000025
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000025
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000025
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000025
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000025
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000020
 			>;
 		};
 
 		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
 			fsl,pins = <
-				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000047
-				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000027
-				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000027
-				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000027
-				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000027
-				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000027
-				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000021
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000040
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x06000020
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x06000020
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x06000020
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x06000020
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x06000020
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x06000020
 			>;
 		};
 
-- 
1.7.9.5

